db/migrate/20170120154842_add_started_on_column_to_miq_tasks.rb

```diff
@@ -0,0 +1,5 @@
+class AddStartedOnColumnToMiqTasks < ActiveRecord::Migration[5.0]
```

Comment:
Would it be better to hold off on this migration until you start working on the code that will populate and display `MiqTask#started_on`?